src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -113,7 +112,8 @@
                 continue;
             }
 
-            if (config.getFormats().isEmpty() || Format.isInsideWhiteList(filePath, config.getFormats())) {
+            if (!config.getFileTypeManager().hasSpecifiedFormats()
+                    || config.getFileTypeManager().isInsideWhitelistedFormats(filePath)) {
```

Comment:
can simply make `isInsideWhitelistedFormats` return true if there is no specified formats.